Sources/BraveShared/BraveStrings.swift

```diff
@@ -266,6 +266,30 @@ extension Strings {
         tableName: "BraveShared", bundle: .module,
         value: "Tab Received",
         comment: "Title for 'Tab Received' Callout, This is shown in the message when a Tab information is received from another sync device. ")
+    public static let p3aCalloutTitle =
+      NSLocalizedString(
+        "callout.p3aCalloutTitle",
+        tableName: "BraveShared", bundle: .module,
+        value: "Help make Brave better.",
+        comment: "Title for p3a (Privacy Preserving Analytics) Full Screen Callout")
+    public static let p3aCalloutDescription =
+      NSLocalizedString(
+        "callout.p3aCalloutDescription",
+        tableName: "BraveShared", bundle: .module,
+        value: "This helps us learn what Brave features are used most often. Change this at any time in Brave Settings under ‘Brave Shields and Privacy’.",
+        comment: "Subtitle - Description for p3a (Privacy Preserving Analytics) Full Screen Callout")
+    public static let p3aCalloutToggleTitle =
+      NSLocalizedString(
+        "callout.p3aCalloutToggleTitle",
+        tableName: "BraveShared", bundle: .module,
+        value: "Share anonymous, private product insights. ",
+        comment: "Title for toggle for enabling p3a (Privacy Preserving Analytics) Full Screen Callout")
+    public static let p3aCalloutLinkTitle =
+      NSLocalizedString(
+        "callout.p3aCalloutLinkTitle",
+        tableName: "BraveShared", bundle: .module,
+        value: "Learn more about our Privacy Preserving Product Analytics (P3A).",
+        comment: "Title for the link that vaviagtes information about p3a (Privacy Preserving Analytics)")
```

Comment:
typo: vaviagtes (not sure what the real word is supposed to be)

Reply:
Fix at https://github.com/brave/brave-ios/pull/6308/commits/31a7834bceee08f82dc986636dc09e06bf38b1f8